etc/cvmfs/domain.d/iris.ac.uk.conf

```diff
@@ -0,0 +1,3 @@
+. ../common.conf
```

Comment:
Please add the 4 lines of prefix before including common.conf that is in all the other domain.d/*.conf files.